.github/workflows/windows-vcpkg.yml

```diff
@@ -326,4 +326,5 @@ jobs:
         tag: ${{ github.event.inputs.release_tag }}
       run: |
         gh release upload "$tag" _build/*.zip
+        gh release upload "$tag" _build/*.exe
```

Comment:
One single line ?
```
gh release upload "$tag" _build/*.zip _build/*.exe
```

From the help page of gh (`gh release upload --help`), it should work
> USAGE
>  gh release upload \<tag\> \<file**s**\>... [flags]